code/workspace/linting-and-testing/.taskcat.yml

```diff
@@ -0,0 +1,10 @@
+project:
+  name: linting-and-testing-workshop
+  package_lambda: false
+  # Update this section with regions you wish to use
+  regions:
+  - us-east-1
+  - us-east-2
+tests:
+  default:
+    template: ./example_vpc_and_security_group.yaml
```

Comment:
this file is missing in `code/solutions/` directory, please add the final working file there

Reply:
Thank you, @rezabekf !  I will also add the `.gitignore` file that is in the relevant `workspace` directory, for consistency.

Reply:
@rezabekf also, since the example `taskcat` configuration above references the `example_vpc_and_security_group.yaml` template, I will add it too to the relevant `solutions` directory.